instrumentation-genai/opentelemetry-instrumentation-openai-v2/src/opentelemetry/instrumentation/openai_v2/patch.py

```diff
@@ -69,14 +80,27 @@ def traced_method(wrapped, instance, args, kwargs):
                 return result
 
             except Exception as error:
+                error_type = type(error).__name__
```

Comment:
```suggestion
                error_type = type(error).__qualname__
```

it should be fully qualified and consistent between spans and metrics